instruqt/proxy/03-app-install/assignment.md

```diff
@@ -0,0 +1,74 @@
+---
+slug: app-install
+id: zfxhfqtljsrb
+type: challenge
+title: Kots application install with proxy config
+teaser: Install kots application via proxy server
+notes:
+- type: text
+  contents: |
+    # Install kots application using a proxy server
+tabs:
+- title: Host
+  type: terminal
+  hostname: isolated-host
+- title: Vendor
+  type: website
+  url: https://vendor.replicated.com
+  new_window: true
+- title: KotsAdm
```

Comment:
You should use the official name "Admin Console" for the tab and in your text.

Reply:
This is probably one of the things to add to a list of things that need to be standardised across all the tracks.  Should probably have a cross track pull request for this kind of thing